app/policies/g_suite_account_policy.rb

```diff
@@ -26,7 +26,7 @@ def update?
   end
 
   def destroy?
-    user.admin?
+    admin_or_manager?
```

Comment:
```suggestion
    user.admin
```

This isn't the policy we should be editing - lowk my fault - Please edit the policy statement on `g_suite_accounts/_panel.html.erb` line 67 to use `!policy(al).destroy?` instead of `!policy(account).destroy?`